doc/TechnicalOverview.md

```diff
@@ -149,8 +149,7 @@ spikes are transferred from one end to another. Both ends, publishers and
 subscribers, connect through a given URL schema.  In the case of spike
 streaming, the URL schema is "monsteer://". If auto-dIscovery is enabled giving
 this schema is sufficient for both ends to connect, otherwise it is also
-possible to hardcode the host and
-its port for the connection.
+possible to hardcode the host and its port for the connection.
```

Comment:
Yes, I looked at the documentation, and I wasn't sure what to change. There are no big conflicts with this change, but it could be clarified, explaining how ZEQ_SESSION can be used for filtering. I am also concerned that about the fact that a single URI is used both for the plugin selection and for the ZMQ connection, which requires all these tricks of replacing the scheme here and there.